tests/test_url_config.py

```diff
@@ -344,6 +344,14 @@ def test_global_version_preprocessor(self):
         self.st.set_config({'test_config': 'fake://url?version=global_v1'})
         p = self.st.get_single_plugin(nt_test_run_id, 'test_data')
         self.assertEqual(p.test_config, 'fake://url?version=v0')
+
+    def alphabetize_url_kwargs(self):
```

Comment:
Why is alphabetize_url_kwargs a method of URLConfig? keep it in the root of the url_config.py file and import it directly from there.